src/translations/strings.yaml

```diff
@@ -852,6 +852,7 @@ inAppMessaging:
   searchBarPlaceholderText: Search messages
   emptyStateTitle: No messages
   emptyStateDescription: You’re up to date!
+  dateTimeYesterday: Yesterday
```

Comment:
wondering also why this was added?

Reply:
It's a string from `src/shared/translations/strings.yaml`, which is removed in this PR.